README.md

```diff
@@ -1,261 +1,42 @@
- * ---------------------------  pushgateway  ---------------------------
- * Step 1: download pushgateway binary from https://prometheus.io/download/#pushgateway
- * Step 2: run push gateway at localhost, set port 9091 using: './pushgateway --web.listen-address=0.0.0.0:9091'
- * Step 3: open a browser and access localhost:9091/
- * ---------------------------  prometheus itself  ---------------------------
- * // TODO: current promethes was not connected with the push-gateway
- * Step 4: download prometheus from https://prometheus.io/download/ ; the tar file contains an executable 'prometheus'
- * Step 5: config 'prometheus.yml' (more info: https://prometheus.io/docs/prometheus/latest/configuration/configuration/)
- * Step 6: start the prometheus with './prometheus --config.file=prometheus.yml'
- * ---------------------------  prometheus cpp client:  ---------------------------
- * The prometheus cpp is expected to be built through MakeFile's git submodule (`git submodule update`). 
- 
- Run instructions below if makefile fails
- * Step 7: mkdir -p submodules/prometheus-cpp/
- * Step 8: in the directory, git clone https://github.com/jupp0r/prometheus-cpp.git
- * Step 9: build prometheus_cpp client following the Building section and 'via CMake' sub-section at
- *         https://github.com/jupp0r/prometheus-cpp
- */
-
-```
-make pdemo
-# replace all of the following param for a different pushgateway setup
-# make pdemo PUSHGATEWAY_NAME=pushgateway PUSHGATEWAY_IP=0.0.0.0 PUSHGATEWAY_PORT=9091
-```
-
-Google how to install other libs for your system.
-
-# Run taraxa-node tests:
-Tests are under core_tests/
-```
-make run_test
-```
-# Run taraxa full node
-To run full node requires one configuration file (in JSON format). 
-
-```
-make main
-```
-use `./build/main --help` to check options.
-
-Run example:
-```
-./build/main --conf_taraxa ./core_tests/conf_taraxa1.json -v 1 --log-filename mylog --log-channels FULLND
-```
-The example run taraxa node with _conf_taraxa1.json_ setting. Reports error and warning message (verbosity 1) in log channel _FULLND_. It also pipe message to _mylog_ file.
-
-Please change ip address, binding port, number of threads, etc, in the configuration files accordingly.
-
-Verbosity setting (-v):
-
-0: Error, 1: Warning (default), 2: Info, 3: Debug, 4: Trace 
-
-## Build debug executable:
-
-```
-make main DEBUG=1
-```
-
-## Docker
-
-Docker has to be installed. If it is not, visit this [web](https://docs.docker.com/install/).
-
-### Building Docker Image
-
-```bash
-$ docker build -t taraxa-node -f dockerfiles/Dockerfile .
-```
+Just want to get up and running quickly? We have pre-built docker images for your convenience. 
+More details are in our [quickstart guide](doc/quickstart_guide.md).
 
-*The building process takes a lot of time, especially during its first execution.*
+# Downloading
 
-### Check Docker images
+Latest release can be always found [here](https://github.com/Taraxa-project/taraxa-node/releases).
```

Comment:
Shouldn't we point to docker hub?

Reply:
yeah I guess so, I would like to have .gz file with taraxad binary and config that people can run directly on ubuntu without docker on this release page, but docker hub should be added here too.  You could perhaps provide these info as I am really sure where we store it and what names are the images going to have etc...

Reply:
https://hub.docker.com/r/taraxa/taraxa-node
This is the image we publish with jenkins.

As for having the .gz, great we can add them later on